docs/source/install.rst

```diff
@@ -141,11 +141,11 @@ The :file:`/etc/pcocc/batch.yaml` configuration file contains configuration pert
      etcd-protocol: http
      etcd-auth-type: password
 
-If you enabled TLS, select the *https* etcd-protocol and define the **etcd-ca-cert** parameter to the path of the CA certificate created for etcd (see :ref:`Deploy a secure etcd cluster <etcd-production>`).
+If you enabled TLS, select the *https* etcd-protocol and define the ``etcd-ca-cert`` parameter to the path of the CA certificate created for etcd (see :ref:`Deploy a secure etcd cluster <etcd-production>`).
 
 The sample network configuration in :file:`/etc/pcocc/networks.yaml` defines a single Ethernet network named *nat-rssh*. It allows VMs of each virtual cluster to communicate over a private Ethernet network and provides them with a network gateway to reach external hosts. Routing is performed by NAT (Network Address Translation) using the hypervisor IP as source. It also performs reverse NAT from ports allocated dynamically on the hypervisor to the SSH port of each VM. DHCP and DNS servers are spawned for each virtual cluster to provide network IP addresses for the VMs. For a more detailed description of network parameters, please see :ref:`pcocc-resources.yaml(5)<networks.yaml>`.
 
-Most parameters can be kept as-is for the purpose of this tutorial, as long as the default network ranges do not conflict with your existing IP addressing plan. The **host-if-suffix** parameter can be used if compute nodes have specific hostnames to address each network interface. For example, if a compute node known by Slurm as computeXX can reached more efficiently via IPoIB at the computeXX-ib address, the **host-if-suffix** parameter can be set to *-ib* so that the Ethernet tunnels between hypervisors transit over IPoIB. Raising the MTU may also help improve performance if your physical network allows it.
+Most parameters can be kept as-is for the purpose of this tutorial, as long as the default network ranges do not conflict with your existing IP addressing plan. The ``host-if-suffix`` parameter can be used if compute nodes have specific hostnames to address each network interface. For example, if a compute node known by Slurm as computeXX can reached more efficiently via IPoIB at the computeXX-ib address, the ``host-if-suffix`` parameter can be set to *-ib* so that the Ethernet tunnels between hypervisors transit over IPoIB. Raising the MTU may also help improve performance if your physical network allows it.
```

Comment:
The formatting is as intended here for host-if-suffix. However, we could use backticks for computeXX.